src/modules/configs/config.py

```diff
@@ -0,0 +1,48 @@
+import os
+from json import loads
+from typing import Union
+
+from jsonschema import validate, ValidationError
+
+from ingestation.modules.global_vars import APPLICATION_DIR
+
+
+def validate_json(conf: dict, schema_file) -> None:
+    validate(instance=conf, schema=getattr(schema_file, "schema"))
+
+
+class Conf:
+    """Class for parsing configuration files"""
+
+    def __init__(self, conf_path: Union[bool, str], conf_type: str, schema_file):
+        self.conf_type = conf_type
+        self.conf_path = self.get_conf_path(conf_path)
+        self.schema_file = schema_file
+
+    def parse_conf(self) -> Union[bool, dict]:
+        conf = self.load_conf()
+
+        try:
+            validate_json(conf, self.schema_file)
+        except ValidationError as ex:
```

Comment:
Imo `self.logger.error(exc)` and then logging critical (using `ColourFormatter`) looks better. 
![image](https://github.com/LegenJCdary/ingeSTation/assets/75223465/5fa9fcf3-f136-41fb-918a-68358d82db43)

Reply:
Will try to use the Loggers class

Reply:
Just pass from `main` to `MergedConf` and then to other conf classes.